kedro-telemetry/kedro_telemetry/plugin.py

```diff
@@ -137,12 +141,15 @@ def after_context_created(self, context):
 
 
 def _get_project_properties(
-    hashed_username: str, project_metadata: ProjectMetadata
+    hashed_username: str, project_metadata: Optional[ProjectMetadata] = None
 ) -> Dict:
 
-    hashed_package_name = _hash(project_metadata.package_name)
-    hashed_project_name = _hash(project_metadata.project_name)
-    project_version = project_metadata.project_version
+    if project_metadata:
+        hashed_package_name = _hash(project_metadata.package_name)
+        hashed_project_name = _hash(project_metadata.project_name)
+        project_version = project_metadata.project_version
+    else:
+        hashed_package_name = hashed_project_name = project_version = "undefined"
```

Comment:
As Ivan mentioned yesterday, it may be possible to get these metadata via `kedro.framework.project`. `package_name` and `project_name` are almost always the same.

Another out-of-scope question is, does the `KedroCLITelemetry` work with the packaged project then?